vqpy/backend/operator/vobj_filter.py

```diff
@@ -0,0 +1,84 @@
+from vqpy.backend.operator.base import Operator
+from vqpy.backend.frame import Frame
+from typing import Callable, Union, Optional, List
+
+
+class VObjFilter(Operator):
+    def __init__(self,
+                 prev: Operator,
+                 condition_func: Union[Callable, str, List[str]],
+                 filter_index: int = 0,
+                 property_name: Optional[str] = None,
+                 ):
+        """
+        Filter vobjs based on the condition_func.
+        :param prev: previous operator
+        :param condition_func: a callable function that takes in a vobj
+            and returns a bool value.
+            If condition_func is a string or a list of string, it should be
+            one or more class name.
+            The vobjs with the class name(s) will be filtered.
+        :param filter_index: the index of the filter.
+            If the index is already used, raise ValueError.
+        :param property_name: the name of the property on vobj.
+            If condition_func is a callable function, property_name must be
+            provided.
+        """
+        self.condition_func = condition_func
+        self.filter_index = filter_index
+        self.property_name = property_name
+        super().__init__(prev)
+
+    def _update_filtered_class(self, class_name, frame: Frame):
+        if class_name in frame.filtered_vobjs[self.filter_index]:
```

Comment:
Is it possible to have two `vobj_filter`s both having the same `filter_index` and and `class_name`